docs/getting-started/user-guides/using-paymetheus.md

```diff
@@ -81,37 +82,40 @@ the outcome of any vote. By spreading tickets around pools, it makes the network
-address) must be given to the pool.
+There's a fair bit of information here, so we'll go through each of the options.
+
+* **Ticket difficulty** - The current price of a ticket.
+* **Blocks until retarget** - When this reaches 0, a new ticket price is calculated.
+* **Source account** - This is the account that will purchase the tickets and receive the reward.
+* **Tickets to purchase** - The number of tickets to purchase.
+* **Ticket fee (DCR/kB)** - Tickets are entered into the voting pool by order of their fee. In times of demand,
+                        you will need to increase this value in order to have your tickets accepted.
+						You can view current ticket fees [here](https://www.dcrstats.com).
+* **Split fee (DCR/kB)** - Sometimes your wallet will need to use multiple transactions to buy tickets. This can
+                       be left at 0.01. It does not affect your chances of buying tickets or voting with them.
+* **Expiry (blocks)** - Often ticket fees will increase during a window and you may be stopped out by higher fees. By setting an
+					expiry, tickets that are not mined in the given number of blocks are cancelled so you can try again
+					with higher fees if you wish. If this is empty, they will not expire until the end of the window.
+* **Stake pool preference** - Automate setup with PoS pools. See below for more information.
+* **Voting address** - The Decred address that will do the voting. Solo and custom pool miners only.
+* **Pool fee address** - For those using a custom pool.
+* **Pool fees (%)** - For those using a custom pool.
 
-<i class="fa fa-exclamation-triangle"></i> **On mainnet, the address will begin with the letters Ds.**
-
-![Generate pubkey address](/img/Paymetheus-generate-pubkey-address.png)  
-
-<i class="fa fa-exclamation-triangle"></i> **On mainnet, the public key address will begin with the letters Dk.**
-
-Copy and paste the public key address into the stake pool's submit address form
-and click the submit button.  Your multisig script and P2SH address will be
-generated and you will be redirected to the tickets page.
-
-Next, import the pool's 1-of-2 multisignature script (on the Scripts tab).  This allows the funds to
-be recovered in case of pool failure.
-
-![Importing a script](/img/Paymetheus-import-script.png)  
+![Purchasing tickets](/img/Paymetheus-ticket-purchasing.png)  
 
-Finally, tickets can be purchased.  Select the "Using a pool?" checkbox and copy
-the values from the pool.  When all fields are set and if there is enough input
-value is available, the purchase button will become enabled and tickets may be
-purchased.
+To easily set up ticket purchasing for a stake pool, click the 'Manage pools button'. If you haven't already,
+you'll need to register with a stake pool (see above). Once you've registerd log in and look for your API key and copy it.
+Select the pool you just registered with from the drop down. Paste the key into the 'API key' box, then click outside the
```

Comment:
at least that is the way it should be working, although we might not be updating until focus is left.. hmm...